File: BlackjackHusofication.Angular.UI/src/models/balanceInfo.ts
import { DealerInfo } from "./dealerInfo";
import { HusokaInfo } from "./husokaInfo";
import { PlayerInfo } from "./playerInfo";

export class BalanceInfo{
    playerInfos : PlayerInfo [] = [];
    dealerInfo : DealerInfo
    husokaInfo : HusokaInfo;
}

File: BlackJackHusofication.Model/Models/Info/BalanceInfo.cs
namespace BlackJackHusofication.Model.Models.Info;

public class BalanceInfo
{
    public List<PlayerInfo> PlayerInfos { get; set; } = [];
    public DealerInfo DealerInfo { get; set; }
    public HusokaInfo HusokaInfo { get; set; }
}
